pkg/codegen/internal/test/testdata/external-resource-schema/dotnet/ArgFunction.cs

```diff
@@ -26,6 +38,16 @@ public ArgFunctionArgs()
         }
     }
 
+    public sealed class ArgFunctionOutputArgs
```

Comment:
*OutputArgs naming convention is something we chose here, can chose differently.